types/mui-datatables/mui-datatables-tests.tsx

```diff
@@ -45,6 +48,12 @@ const MuiCustomTable: React.FC<Props> = props => {
         {
             name: 'amount',
             label: 'Amount',
+            options:{
+                customHeadLabelRender: (dataIndex: number, rowIndex: number) => {
+                    console.log(`custom header label render for dataIndex ${dataIndex} and rowIndex ${rowIndex}`)
+                    return (<p>Some customize Header</p>);
```

Comment:
```suggestion
                    return <p>Some customize Header</p>;
```

No need for braces here